packages/rpc-graphql/src/context.ts

```diff
@@ -195,13 +189,11 @@ async function resolveTransaction(
 export function createSolanaGraphQLContext(rpc: Rpc<SolanaRpcMethods>): RpcGraphQLContext {
     const cache = createGraphQLCache();
     return {
+        block: new DataLoader(createBlockBatchLoadFn(rpc), { cacheKeyFn: fastStableStringify }),
```

Comment:
This `cacheKeyFn` is actually insufficient, because you want to apply _defaults_ before creating the cache key.

For instance, these should produce the _same_ cache key:

* `{slot: 123n, commitment: 'finalized'}`
* `{slot: 123n}`

…because `finalized` is the server default.

I'll leave the creation of this `normalizeConfig` method to the reader (that's you!)

Reply:
Yeah, this is a great call-out. I'll add to my active stack!